package.json

```diff
@@ -1,26 +1,27 @@
 {
   "name": "native-template",
-  "version": "6.3.5",
+  "version": "7.0.0-alpha.1",
   "private": true,
   "scripts": {
-    "postinstall": "patch-package && jetify",
+    "postinstall": "node patches/apply-patches.js && jetify",
     "configure": "native-mobile-toolkit configure --config-path='./config.json' --verbose"
   },
   "dependencies": {
-    "@mendix/native": "~1.0.0",
+    "@mendix/native": "alpha",
```

Comment:
can this package name be more descriptive? e.g. `2.0.0-alpha`

Reply:
Why? Thiamin using npm tag, this is not a version which means it will always take latest alpha